common/models/transducer/loss.py

```diff
@@ -4,7 +4,7 @@
 
 def rnnt_loss(source, **_kwargs):
   """
-  Computes the RNN-T loss function.
+  Computes the RNN-T loss function. Cuda implementation.
```

Comment:
This is wrong. It uses HawkAaronWarpTransducer (this is some information which you might add here). HawkAaronWarpTransducer supports both CPU and GPU (CUDA).

Reply:
You might add that HawkAaronWarpTransducer is a native TF kernel implementation.